db/seed.js

```diff
@@ -0,0 +1,61 @@
+export const addTables = db => {
+    db.transaction( tx => {
+        tx.executeSql(addMonstersTable);
+        tx.executeSql(addSessionsTable);
+        tx.executeSql(addActiveMonstersTable);
+    },transactionError,transactionSuccess)
+};
+
+
+export const seedMonstersTable = db => {
+    db.transaction( tx => {
+        addTestMonster(tx,'Harold');
+    },transactionError,transactionSuccess)
+};
+
+const transactionError = (err) => console.log("ERROR",err);
+const transactionSuccess = () => console.log("SUCCESS");
+
+
+const addMonstersTable = `
+    CREATE TABLE IF NOT EXISTS monsters (
+        id INTEGER PRIMARY KEY,
+        name VARCHAR(20),

```

Comment:
Is the name limited to 20 characters and the notes limited to 255? Both of those could be potentially important limiters. The name is most likely fine, but keep in mind that creatures in dnd can have pretty long names. Additionally, since I might put attacks or other important information in the notes, it definitely needs to be longer. You will probably need to change the data structure holding it, since VARCHAR is limited.

Reply:
ah, Good point!